lib/linter-views.coffee

```diff
@@ -34,9 +34,8 @@ class LinterViews
 
     @panel.id = 'linter-panel'
     @updateTabs()
-
-  getMessages: ->
-    @messages
+    @linter.subscriptions.add @linter.onDidChangeMessages =>
```

Comment:
Wait, what? Why are you adding a `Disposable` to `@linter` instead of keeping it local?